securedrop-admin

```diff
@@ -46,8 +46,21 @@ def activate_venv():
 def sdconfig(args):
     """Configure SD site settings"""
     activate_venv()
-    subprocess.check_call([os.path.join(ANSIBLE_PATH,'configure')],
-                            cwd=ANSIBLE_PATH, env=os.environ.copy())
+
+    ansible_cmd = [
+        'ansible-playbook',
+        # Passing an empty inventory file to override the automatic dynamic
+        # inventory script, which fails if no site vars are configured.
+        '-i', '/dev/null',
+        os.path.join(ANSIBLE_PATH, 'securedrop-configure.yml'),
```

Comment:
The trailing comma is unnecessary, but harmless. It is ignored by the Python interpreter. I just have noticed you using it in your code often, and just wanted to comment. I'm not sure if it's been done intentionally or not.